source/sass/patterns/atoms/image.scss

```diff
@@ -0,0 +1,6 @@
+.image {
```

Comment:
Needs to have block spacing like https://github.com/libero/pattern-library/blob/ed02233a65ab61e0c452161a80a705a03647f219/source/sass/patterns/molecules/figure.scss#L6 (set to 0/`$baselinegrid-space-extra_small` when a `figure__asset`).